src/azure-cli/azure/cli/command_modules/acr/linter_exclusions.yml

```diff
@@ -11,5 +11,7 @@ acr update:
         allow_trusted_services:
             rule_exclusions:
             - option_length_too_long
-
+acr connected-registry update:
+    rule_exclusions:
+    - missing_command_test_coverage
```

Comment:
Please add some test cases for this command instead of excluding it in tests

Reply:
I  ALREADY DID. That's what I've been asking about for a weeks now.

Reply:
Check the test file. The update test has been there for years now...